test/tabs_test.js

```diff
@@ -93,6 +93,7 @@ describe('Tabs', function() {
 
   describe('selecting new tab (2)', function() {
     before(function(done) {
+      const browser = new Browser();
```

Comment:
This shouldn't make any difference, since in the next line we're closing all the tabs and essentially starting from a browser with zero tabs … unless there's a bug in the code.